src/pages/use-cases/cni.jsx

```diff
@@ -0,0 +1,105 @@
+import IntroSection from 'components/pages/use-cases/intro-section';
+import VideoFeatureSection from 'components/pages/use-cases/video-feature-section';
+import AWSLogo from 'icons/logo-aws.inline.svg';
+import AzureLogo from 'icons/logo-azure.inline.svg';
+import GoogleLogo from 'icons/logo-google.inline.svg';
+import CNIImage1 from 'images/pages/usecase/cni-1.png';
+import CNIImage2 from 'images/pages/usecase/cni-2.png';
+import CNIImage3 from 'images/pages/usecase/cni-3.png';
+import MainLayout from 'layouts/main/main';
+
+const introContent = {
+  title: 'High Performance Cloud Native Networking (CNI)',
+  category: 'Networking',
+  tagline: 'Enhance the speed and efficiency of your Kubernetes and cloud native networks',
+  subHeading: 'How can I have scalable and consistent networking across clouds?',
+  description:
+    'There are dozens of CNIs available for Kubernetes but, their features, scale, and performance vary greatly. Many of them rely on a legacy technology (iptables) that cannot handle the scale and churn of Kubernetes environments leading to increased latency and reduced throughput. Most CNIs also only offer support for L3/L4 Kubernetes network policy but little beyond. Many Cloud Provider have their own custom CNIs which results in operational complexity for customers operating in multi-cloud environments.',
+  isImage: false,
+  videoSrc: 'https://www.youtube.com/embed/ELyib78vjRY',
+};
+
+const sectionContent1 = {
+  title: 'High Performance Cloud Native Networking CNI',
+  description:
+    'Cilium’s control and data plane has been built from the ground up for large-scale and highly dynamic cloud native environments where 100s and even 1000s of containers are created and destroyed within seconds. Cilium’s control plane is highly optimized, running in Kubernetes clusters with 1,000s of nodes and 100K pods. Cilium’s data plane uses eBPF for efficient load-balancing and incremental updates, avoiding the pitfalls of large iptables rulesets. ',
+  imageSrc: CNIImage1,
+  imageAlt: 'cilium cni illustration',
+};
+
+const sectionContent2 = {
+  title: 'Scalability',
+  description:
+    "Cilium is built to scale. Whether you're running a few nodes or managing a cluster with thousands, Cilium can handle it. Cilium’s eBPF-powered networking is optimized for large scale operations. This means you can grow your operations without worrying about the network becoming a bottleneck.",
+  imageSrc: CNIImage2,
+  imageAlt: 'cilium scalability illustration',
+  whiteBackground: true,
+};
+
+const sectionContent3 = {
+  title: 'Security',
+  description:
+    "Cilium doesn't just excel in performance—it also brings robust security features. With identity-based security that goes beyond traditional IP address-based ACLs, Cilium provides fine-grained policy enforcement. This allows you to control who can talk to whom, reducing the attack surface and helping secure your cloud native applications.",
+  imageSrc: CNIImage3,
```

Comment:
This is currently an image for Tetragon. It should be updated to a network security image